### PR TITLE
Allow alternative Solr Schema for Solr Highlighting of File Content

### PR DESCRIPTION
**Pull request by fergusonna**

### CODE REVIEW

File: vagrant/provisioning/roles/solr/tasks/configset.yml [hunk 1]
```diff
@@ -51,6 +51,15 @@
     dest: "{{ root_folder }}/app/solr/server/solr/configsets/arkcase/conf/"
     mode: 0644
   when: "schema_xml.stat.exists == false or 'AFDP' not in schema_contents.stdout"
+
```
Comment (david-oc-miller) [inline vagrant/provisioning/roles/solr/tasks/configset.yml:54]: On the previous task ("copy schema.xml if needed") you can add another clause to the `when`: `... and not (enable_solr_snippets | default(false))`.

File: vagrant/provisioning/roles/solr/tasks/configset.yml [hunk 2]
```diff
@@ -51,6 +51,15 @@
...
+- name: apply solr-snippet-enabled.xml if needed
+  become: yes
+  become_user: solr
+  copy:
+    src: solr-snippet-enabled.xml
+    dest: "{{ root_folder }}/app/solr/server/solr/configsets/arkcase/conf/schema.xml"
+    mode: 0644
+  when: "enable_solr_snippets == true"
```
Comment (david-oc-miller) [inline vagrant/provisioning/roles/solr/tasks/configset.yml]: Maybe just `enable_solr_snippets | default(false)`

File: vagrant/provisioning/roles/solr/tasks/configset.yml (hunk 1 above)
Comment (fergusonna) [reply vagrant/provisioning/roles/solr/tasks/configset.yml:54]: Updated

File: vagrant/provisioning/roles/solr/tasks/configset.yml (hunk 2 above)
Comment (fergusonna) [reply vagrant/provisioning/roles/solr/tasks/configset.yml]: Updated

File: vagrant/provisioning/roles/solr/tasks/configset.yml (hunk 1 above)
Comment (david-oc-miller) [reply vagrant/provisioning/roles/solr/tasks/configset.yml:54]: I added extra parentheses to make the order of operations explicit; and also fixed the octal permission flags due to a warning from ansible-lint.

**Review by david-oc-miller — APPROVED** [review]

**Pull request merged**